core/src/test/java/apoc/temporal/TemporalProceduresTest.java

```diff
@@ -64,6 +64,13 @@ public void shouldFormatDuration() throws Throwable
         String output = TestUtil.singleResultFirstColumn(db, "RETURN apoc.temporal.format( duration('P0M0DT4820.487660000S'), \"HH:mm:ss.SSSS\" ) as output");
         assertEquals("01:20:20.4876", output);
     }
+    
+    @Test
+    public void shouldFormatDurationGreaterThanADay() {
+        String output = TestUtil.singleResultFirstColumn(db, "WITH duration.between(datetime(\"2017-06-02T18:40:32\"), datetime(\"2019-07-03T19:41:33\")) AS duration\n" +
+                "RETURN apoc.temporal.formatDuration(duration, \"yy 'years' MM 'months' dd 'days' - HH:mm:ss\") AS value");
+        assertEquals("02 years 01 months 01 days - 01:01:01", output);
+    }
 
     @Test
```

Comment:
It would probably be nice to have more tests in the future. For example, I don't think the test below would pass.

```
    @Test
    public void test() {
        String output = TestUtil.singleResultFirstColumn(db, "WITH duration.between(datetime(\"2019-07-03T18:40:32\"), datetime(\"2019-07-03T19:41:33\")) AS duration\n" +
                "RETURN apoc.temporal.formatDuration(duration, \"yy 'years' MM 'months' dd 'days' - HH:mm:ss\") AS value");
        assertEquals("00 years 00 months 00 days - 01:01:01", output);
    }
```